tests/benchmarks/multicore/idle/boards/nrf54h20dk_nrf54h20_cpuapp_s2ram.overlay

```diff
@@ -16,6 +16,7 @@
                        compatible = "zephyr,power-state";
                        power-state-name = "suspend-to-ram";
                        min-residency-us = <800000>;
+                       exit-latency-us = <1000>;
```

Comment:
Where does this value come from? Have anyone measured the real exit latency value for this CPU in this sleep state?

Reply:
Suggested by @meijemac.
I do not have any measurements for that (I even do not know how to make them).

Reply:
Then we need at least a comment here like:

```suggestion
                       /* TODO: replace with a busy loop in S2RAM entering procedure; replace 1000 with real value here */
                       exit-latency-us = <1000>;
```

However, I would suggest implementing the busy loop in the first place instead of making this temporary solution with a TODO comment.

Reply:
Comment added, thanks.